recipes/plantcv/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+sed -i'' -e 's/versioneer.get_version()/"3.0.3"/g' setup.py
+sed -i'' -e 's/cmdclass=versioneer.get_cmdclass(),//g' setup.py
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
can we use $PYTHON pip install here? This could potentially remove setuptools form the host environment and you need to add `pip` instead.